src/pedal/mod.rs

```diff
@@ -0,0 +1,26 @@
+mod transparent;
+
+pub use transparent::Transparent;
+
+use std::fmt::{Debug, Display};
+
+pub type Boxed = Box<dyn Pedal>;
```

Comment:
A `Box` puts its value on the heap and basically acts as a pointer. The reason I needed that behavior for pedals was that I need to have a `Vec` of `Pedal`s, but since `Pedal` isn't a concrete type (and therefore doesn't have a known size, or the same size for every element of the `Vec`), that's not possible. But since a `Box` is just a pointer, the size is known, so it's allowed as the element type of `Vec`.